clang/lib/Sema/SemaDeclAttr.cpp

```diff
@@ -2064,9 +2064,18 @@ bool Sema::CheckAttrNoArgs(const ParsedAttr &Attrs) {
   return false;
 }
 
+static bool isAttrValidForSYCLDevice(ASTContext &Context, const ParsedAttr &AL)
```

Comment:
Minor comment: Please use clang-format, or fix this line manually to comply with the coding std. '{' must not be in new line:
If should be something like:
  `static bool isAttrValidForSYCLDevice(ASTContext &Context, 
                                                              const ParsedAttr &AL) {`
You would also need to split this line to avoid exceeding 80 syms in a line.

Reply:
Will change.  thanks!

Reply:
Changed